lite-rpc/src/bridge.rs

```diff
@@ -597,4 +615,41 @@ impl LiteRpcServer for LiteBridge {
 
         Ok(())
     }
+
+    async fn get_latest_account_priofees(
+        &self,
+        account: String,
+        method: Option<PrioritizationFeeCalculationMethod>,
```

Comment:
would not use an enum+optional. couldn't you just define enum with appropriate variants?